### PR TITLE
add SensorActivity.kt

### PR DESCRIPTION
**Pull request by AliaksandrG**

### CODE REVIEW

File: app/src/main/java/com/example/myapplication/SensorActivity.kt [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+import android.view.View
+import android.widget.TextView
+import androidx.appcompat.app.AppCompatActivity
+import androidx.core.content.ContextCompat
+
+class SensorActivity : AppCompatActivity(), SensorEventListener {
+
+    var tvSensor: TextView? = null
+    var sensorManager: SensorManager? = null
+    var sensor: Sensor? = null
```
Comment (andrewKarachun0304) [inline app/src/main/java/com/example/myapplication/SensorActivity.kt:18]: use lateinit

File: app/src/main/java/com/example/myapplication/SensorActivity.kt [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+        super.onResume()
+        sensorManager!!.registerListener(this, sensor, SensorManager.SENSOR_DELAY_FASTEST)
+    }
+
+    override fun onSensorChanged(event: SensorEvent) {
+        val currentLux = event.values[0].toString()
+
+        if (event.sensor.type == Sensor.TYPE_LIGHT) {
+            tvSensor!!.text = currentLux
+        }
```
Comment (andrewKarachun0304) [inline app/src/main/java/com/example/myapplication/SensorActivity.kt:44]: don't use assert calls

**Review by andrewKarachun0304 — APPROVED** [review]

**Review by Mikhail-Kh — APPROVED** [review]

File: app/src/main/java/com/example/myapplication/SensorActivity.kt (hunk 1 above)
Comment (AliaksandrG) [reply app/src/main/java/com/example/myapplication/SensorActivity.kt]: done

File: app/src/main/java/com/example/myapplication/SensorActivity.kt (hunk 2 above)
Comment (AliaksandrG) [reply app/src/main/java/com/example/myapplication/SensorActivity.kt]: done

**Review by TMSlebedev — APPROVED** [review]

**Pull request merged**